README.md

```diff
@@ -1,32 +1,31 @@
-## Submitting an Idea
+# How to create and publish an article at Rootstrap blog page
 
-For the management of proposals and new ideas for blog posts, we use this repo's [issues](https://github.com/rootstrap/blog/issues). If you have an idea for an article, either you want to write it yourself or not, you can create a new issue describing it and tagging it accordingly depending on which category the article would fit in.
+Publishing an article help us to improve Rootstrap brand with quality content on the web. Also writing about a topic, enhance your knowledge of it. If you want to create and publish an article, please follow these simple steps:
 
-To make this process easier, we provide an issue template where we ask you for some basic information on your proposal: what’s the motivation for writing about the topic, some notes on the idea itself and possible keywords to identify the article with.
+1. If you have an idea, [submit it](https://github.com/rootstrap/blog/blob/master/SUBMIT-AN-IDEA.md) as an [issue](https://github.com/rootstrap/blog/issues). Otherwise, you can choose a submitted iddea from this repository issues and assign it to yourself.
+2. Write the article. Please take a look at the [guides](https://github.com/rootstrap/blog/blob/master/GUIDE.md).
+3. [Create a PR to `master`](https://github.com/rootstrap/blog/blob/master/CREATE-PR.md) and set the [corresponding labels](https://github.com/rootstrap/blog/blob/master/LABELS.md).
 
-Once you create the issue, you can either assign it to yourself if you’re planning on writing the article, or you can leave it unassigned for anyone interested to pick it up. The blog maintainers will reach out to the reporter and assignee to further refine the idea and eventually decide to include it on a blog release.
+   - Create the `.md` file with the article under the `content` folder with the filename: `<article-title> - <author-name>`.
+   - Suggested name for the branch: `article/<title>`.
 
-**IMPORTANT:** We highly recommend you create an issue before starting to write an article. During the submitting process the idea may change a bit and new ideas may come up, causing the scope of the article to change. This is not to discourage you to start writing articles, but to be aware that changes may happen when going through the pipeline.
+4. Ask for reviewers in the `technical-blog` channel. Try to have at least two approves.
```

Comment:
Maybe here we can also say that it is good that the reviewers know about the topic of the article, like the Notion page says.

Reply:
What do you think about: `Ask for reviewers in the `technical-blog` channel. If it's possible, a good practice is to choose reviewers that have knowledge about the blog topic. Try to have at least two approves.`